{
  "builds": {},
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Mar 19, 2020 10:58:01 AM",
    "reproductionDateEnd": "Mar 19, 2020 11:17:29 AM",
    "machineInfo": {
      "numberCPU": 1,
      "freeMemory": 36545640,
      "totalMemory": 95789056
    },
    "totalDuration": 1164,
    "processDurations": {
      "cloning": {
        "nbSteps": 0,
        "totalDuration": 0,
        "stepNames": [],
        "stepDurations": []
      },
      "building": {
        "nbSteps": 3,
        "totalDuration": 3,
        "stepNames": [
          "ComputeSourceDir",
          "ComputeTestDir",
          "CheckoutPatchedBuild"
        ],
        "stepDurations": [
          0,
          3,
          0
        ]
      },
      "testing": {
        "nbSteps": 0,
        "totalDuration": 0,
        "stepNames": [],
        "stepDurations": []
      },
      "fixing": {
        "nbSteps": 1,
        "totalDuration": 106,
        "stepNames": [
          "AstorJKali"
        ],
        "stepDurations": [
          106
        ]
      }
    },
    "projectRootPomPath": "/scratch/dginelli/workspace/repairnator-repairnator-experiments-javaee-samples-javaee7-samples-228543267-20170504-053413_bugonly-firstCommit/pom.xml"
  },
  "repository": {
    "githubId": 0,
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "commits": {},
  "tests": {
    "failingModule": "/scratch/dginelli/workspace/repairnator-repairnator-experiments-javaee-samples-javaee7-samples-228543267-20170504-053413_bugonly-firstCommit/batch/chunk-csv-database",
    "overallMetrics": {
      "numberRunning": 3,
      "numberPassing": 2,
      "numberFailing": 0,
      "numberErroring": 1,
      "numberSkipping": 0,
      "failures": [
        {
          "failureName": "com.sun.jersey.api.container.ContainerException",
          "isError": true,
          "occurrences": 1
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "org.javaee7.batch.chunk.csv.database.BatchCSVDatabaseTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "org.javaee7.batch.chunk.csv.database.BatchCSVDatabaseTest",
        "testMethod": "org.javaee7.batch.chunk.csv.database.BatchCSVDatabaseTest",
        "failureName": "com.sun.jersey.api.container.ContainerException",
        "detail": "exit_code: FAILURE, message: Error occurred during deployment: Exception while preparing the app : Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.6.4.qualifier): org.eclipse.persistence.exceptions.DatabaseException\nInternal Exception: java.sql.SQLException: Error in allocating a connection. Cause: Connection could not be allocated because: java.net.ConnectException : Error connecting to server localhost on port 1527 with message Connection refused (Connection refused).\nError Code: 0. Please see server.log for more details. [status: CLIENT_ERROR reason: Bad Request]",
        "isError": true
      }
    ]
  },
  "projectMetrics": {
    "numberModules": 228,
    "numberPlugins": 0,
    "numberSourceFiles": 742,
    "numberTestFiles": 201,
    "numberLibrariesFailingModule": 111,
    "numberLines": 0
  }
}